text/0002-qri_dataset_definition.md

```diff
@@ -0,0 +1,343 @@
+
+All of the meta fields below must be well-formed valid values. However: _The Meta section of a dataset supports arbitrary metadata_. This means you can place additional values not listed here & qri will store them as-is, without any additional validation.
+
+Another important note here: _qri software doesn't leverage things like identifier, downloadPath, homePath, etc._. Qri considers all fields here _descriptive_ metadata, as opposted to _structural_ metadata. In practice this means qri User interfaces may leverage the meta component for the purposes of _correlation_ and _display_. Information stored in the meta section is _not_ intended for use by machines to interpret the dataset. For example, setting the `identifier` is of no significance to qri, it's included here for interoperability with other specifications like [DCAT](https://www.w3.org/TR/vocab-dcat/#Property:dataset_identifier)
+
+_meta fields:_
+
+| name                  | type          | description   |
+|-----------------------|---------------|---------------|
+| `accessPath`          | `string`      | url or location to access this dataset.   |
+| `accrualPeriodicity`  | `string`      | frequency at which dataset changes. Must be an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Repeating_intervals) repeating duration   |
+| `citations`           | `[]Citation`  | array of assets used to build this dataset   |
+| `contributors`        | `[]User`      | description   |
+| `description`         | `string`      | roughly a paragraph of human-readable text that provides context for the dataset |
+| `downloadPath`        | `string`      | URL or other path string to where to download this dataset   |
+| `homePath`            | `string`      | URL or other path string to a "landing page" resource that explains the dataset  |
+| `identifier`          | `string`      | identifer for the dataset   |
+| `keywords`            | `[]string`    | list of "tags" to connect this dataset with other datasets that carry similar keywords   |
+| `language`            | `[]string`    | array of languages this dataset is written, in [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) format, ordered by most-to-least dominant  |
+| `license`             | `License`     | the legal licensing agreement this dataset is released under  |
+| `title`               | `string`      | title of the dataset |
+| `theme`               | `[]string`      | "categories" this dataset should be filed under. Keywords should draw out specific keywords, theme entires should speak to the broader family of datasets this dataset is part of  |
+| `version`             | `string`      | version identifier string   |
+
+_additional data types:_
+
+#### `Citation`
+
+_example:_
+```json
+  { 
+    "name" : "sean carter",
+    "url" : "https://jayz.com",
+    "email" : "hova@jayz.com"
+  }
```

Comment:
I'm confused why we don't have peerids in citations, and why we don't have peernames in users and citations. 

And I guess why citations are a slice of citations rather than a slice of users.

I can see the logic that a citation may not be a user (peer), but having a way to give credit that will link to another peer will probably be useful in the future, when people work on teams or folks are referencing other qri peers when making datasets.

Reply:
> I'm confused why we don't have peerids in citations, and why we don't have peernames in users and citations.

We actually do have peerID's in Citations, in an ID field that isn't identified here. I'll update.

> And I guess also confused why citations are a slice of citations rather than a slice of users.

Citations can be both a _thing_ and a _person_. One can cite a person saying something, or they can quote a book or text (with a URL) 

In both these cases, we're working within the context of an existing Metadata specification (DCAT & the Dublin Core) and I want to stay as close to it as possible so we get better metadata interop. So it means that we're kinda stuck shoehorning our own user info into an existing pattern.